build.sh

```diff
@@ -1,18 +1,33 @@
+# Remove existing GOV.UK Frontend assets
```

Comment:
This is how the demo creates it assets, but we use Flask Assets to achieve this within the skeleton.

Reply:
Thank you, I have removed this section with #94aa36f1